server/src/graphql/queries/githubInstallationId.ts

```diff
@@ -21,7 +22,21 @@ export const githubInstallationId: QueryResolvers['githubInstallationId'] = asyn
     auth: user.githubAccessToken,
   });
 
-  const installations = await octokit.request('GET /user/installations');
+  let installations;
+
+  try {
```

Comment:
while this is working for this call, it means that we will have to do a try catch in every call that is using the user token. We should abstract the request logic into a function that we can reuse easily )